conans/client/conan_api.py

```diff
@@ -45,10 +45,19 @@
 default_manifest_folder = '.conan_manifests'
 
 
+def get_request_timeout(client_cache):
+    timeout = os.getenv("CONAN_REQUEST_TIMEOUT") or client_cache.conan_config.request_timeout
```

Comment:
This is redundant, ``client_cache.conan_config.request_timeout`` already checks the env-var

Reply:
Actually is the opposite, I should check only the env var. The config var is not updated with the environment.